cicecore/cicedynB/dynamics/ice_dyn_shared.F90

```diff
@@ -1899,7 +1903,7 @@ subroutine viscous_coeffs_and_rep_pressure (strength,  tinyarea, &
       real (kind=dbl_kind), intent(in)::  &  
         Deltane, Deltanw, Deltasw, Deltase  ! Delta at each corner
 
-      logical , intent(in):: capping
+      real(kind=dbl_kind) , intent(in):: capping
```

Comment:
It looks like the intent here is that capping is either c0 or c1 as if it were a logical but by setting it to a real it avoids an if check.  That should be documented in the code and should there be a check added that capping must be either c0 or c1 valued?

Reply:
capping was introduced by @JFLemieux73 when he reorganized the EVP solver, This introduced an if statement within the viscous_coeffs_and_rep_pressure function. To avoid the if statement I replaced it with a c0/c1. It is c1 in the evp and c0 in the vp and set as parameters further up. Currently hard coded. The intent is to replace it with a namelist parameter both in main and in this code

Reply:
Will it always be either 0 or 1?  And will it always be 1 for EVP (and presumably the variants) and 0 for VP?  If so, could it be set based on kdyn at initialization, instead of namelist?